src/storage/size-reporter/SizeReporter.ts

```diff
@@ -0,0 +1,22 @@
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+import type { Size } from './size.model';
+
+/**
+ * A SizeReporter's only purpose (at the moment) is to calculate the size
+ * of a resource. How the size is calculated or what unit it is in is defined by
+ * the class implementing this interface.
+ * One might use the amount of bytes and another might use the amount of triples
+ * stored in a resource.
+ */
+export interface SizeReporter {
+
+  unit: string;
```

Comment:
Should not be in the interface IMHO.

Reply:
It might be usefull for other classes to see what kind of data the SizeReporter will return ( Thinking about some admin interface ).